src/backend/distributed/sql/udfs/stop_metadata_sync_to_node/latest.sql

```diff
@@ -0,0 +1,6 @@
+CREATE FUNCTION pg_catalog.stop_metadata_sync_to_node(nodename text, nodeport integer, clear bool DEFAULT false)
```

Comment:
Wondering whether we should make this true. It seems like a cleaner default behaviour.

Reply:
Makes sense to me. What do you think @onderkalaci ?

Reply:
Yes, I think so